README.md

```diff
@@ -27,17 +27,17 @@ This list contains only stage 1 proposals and higher that have not yet been with
 |   | [Optional catch binding](https://michaelficarra.github.io/optional-catch-binding-proposal/)                 | Michael Ficarra                    | 3 |
 |   | [`function.sent` metaproperty](https://github.com/allenwb/ESideas/blob/master/Generator%20metaproperty.md)  | Allen Wirfs-Brock                  | 2 |
 |   | [`String.prototype.{trimStart,trimEnd}`](https://github.com/sebmarkbage/ecmascript-string-left-right-trim)  | Sebastian Markbage                 | 2 |
-|   | [Class and Property Decorators](http://tc39.github.io/proposal-decorators/)                                 | Yehuda Katz and Brian Terlson      | 2 |
+|   | [Class and Property Decorators](http://tc39.github.io/proposal-decorators)                                  | Yehuda Katz and Brian Terlson      | 2 |
```

Comment:
To properly normalize them, they should all have the URL they end up at after redirects - which means they should all have a slash.